docuploader/__main__.py

```diff
@@ -92,7 +96,11 @@ def upload(
 
     docuploader.log.info("Loading up your metadata.")
     try:
-        shutil.copy(metadata_path, pathlib.Path(documentation_path) / metadata_file)
+        if documentation_path not in metadata_file:
```

Comment:
Do we still need the `try`/`except`?

Reply:
Seems like the code works fine even if I do something like `python3 -m docuploader upload . --staging-bucket test-bucket-pipeline --destination-prefix docfx- --metadata-file docs.metadata`. Removed try/except.